### PR TITLE
Added MOL 19

### PR DESCRIPTION
**Pull request by mjpost**

### CODE REVIEW

File: bin/ingest.py [hunk 1]
```diff
@@ -53,17 +54,30 @@
...
+            else:
+                print(f'Volume {volume_id} has already been inserted into {collection_file}.')
+                print(f'You can append to this volume by passing `--append` (or `-a`) to this script.')
+                print(f'Quitting, since you didn\'t.')
+                sys.exit(1)
+
+            break
+    else:
+        # If no existing volume was found, append the volume
+        # TODO: find correct insertion point in the sequence of existing volumes, instead of appending
```
Comment (knmnyn) [inline bin/ingest.py]: @mjpost Once merged, you may want to make a new issue for this.  Appending probably breaks people expectations of where a volume's metadata should be.

Comment (mjpost) [reply bin/ingest.py]: Agreed. What I've done so far is manually correct it. This script gets me halfway there...

Comment (mjpost) [reply bin/ingest.py]: Okay, I finished this and pushed it up.

**Comment by mjpost** [comment]
Thanks for the review!

**Pull request merged**